department-of-veterans-affairs/caseflow
# APPEALS-57167: Incorrect distribution of CAVC Appeals with Hearing held after Remand

Resolves [APPEALS-57167](https://jira.devops.va.gov/browse/APPEALS-57167)

# Description
Modifies the `cavc_affinity_filter` and `cavc_aod_affinity_filter` for when the lever values are infinite to check whether the case has a hearing held after the previous decision in addition to whether the most recent hearing judge is ineligible. This fixes a bug where if the most recently held hearing was by an ineligible judge the case would be treated strictly as genpop, when it should have been considered to have an affinity to the previous deciding judge.

## Acceptance Criteria

## Testing Plan

1. Login as an active judge user
2. Navigate to their judge assignment queue page
3. Request more cases
4. Verify that cases where the most recent hearing was held by an ineligible judge AND the previous decision was from an active judge AND are within their affinity window are not distributed to a judge who is not the previous deciding judge.
